core/domain/exp_jobs_one_off.py

```diff
@@ -57,6 +57,43 @@
 ])
 
 
+class RemoveDeprecatedExplorationModelFieldsOneOffJob(
+        jobs.BaseMapReduceOneOffJobManager):
+    """Job that sets skill_tags, default_skin, skin_customization fields
+    in ExplorationModels to None in order to remove it from the datastore.
```

Comment:
Please add a newline saying that this job is needed just for the January 2021 release and can be removed after.

Reply:
done